.github/workflows/github-container-registry.yml

```diff
@@ -0,0 +1,169 @@
+##########################################################################
+#
+# WARNING: This file was generated by update.php.
+#
+# You can find the relevant template in the `/templates` folder.
+#
+name: Build test images
+
+on:
+  pull_request:
```

Comment:
Why every PR? Maybe I don't understand what we use GitHub container repository for 🙂

Reply:
We use GitHub container registry to host generated images for testing changes made in PRs before merging and pushing the containers out to Docker Hub. So this workflow should only run for pull requests.